client/src/app/pages/applications/applications-table-analyze/applications-table-analyze.tsx

```diff
@@ -438,20 +440,32 @@ export const ApplicationsTableAnalyze: React.FC = () => {
     : [];
   const applicationDeleteDropdown = applicationWriteAccess
     ? [
-        <DropdownItem
-          key="manage-applications-bulk-delete"
-          isDisabled={
+        <ConditionalTooltip
+          isTooltipEnabled={
             selectedRows.length < 1 ||
-            selectedRows.filter(
+            selectedRows.some(
```

Comment:
I think we want to delete the app even though it is not selected?

Reply:
Oh oops this is for the toolbar bulk delete.

Reply:
Ah yeah!

Reply:
Moved my comment to the correct line.

Reply:
@ibolton336, this is the block for bulk-delete. 
So if at least one of the selected application is involved in a migration wave then delete is disable.
So some() still works here.

Reply:
You've got it! Just need to update line 664 to check all applications rather than selected.